scripts/technical-debt-metrics.sh

```diff
@@ -12,18 +12,8 @@ IFS=$'\n\t'
 # the script takes two optional arguments `<optCurrCommit> <optReferenceCommit>`
 # and tallies the same technical debts on `<optCurrCommit>` using `<optReferenceCommit>`
 # as a reference.
-
-if [ -n "${1}" ]; then
-  currCommit="${1}"
-else
-  currCommit="$(git rev-parse HEAD)"
-fi
-
-if [ -n "${2}" ]; then
-  refCommit="${2}"
-else
-  refCommit="$(git log --pretty=%H --since="$(date -I -d 'last week')" | tail -n -1)"
-fi
+currCommit=${1:-$(git rev-parse HEAD)}
+refCommit=${2:-$(git log --pretty=%H --since="$(date -I -d 'last week')" | tail -n -1)}
```

Comment:
```suggestion
currCommit=${1:-"$(git rev-parse HEAD)"}
refCommit=${2:-"$(git log --pretty=%H --since="$(date -I -d 'last week')" | tail -n -1)"}
```